tanner/postgres_client.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+import logging
+
+import aiopg
+
+from tanner.config import TannerConfig
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PostgresClient:
+    def __init__(self):
+        self.logger = logging.getLogger("tanner.db_helper.postgres")
+
+        self.host = TannerConfig.get("POSTGRES", "host")
+        self.port = TannerConfig.get("POSTGRES", "port")
+        self.user = TannerConfig.get("POSTGRES", "user")
+        self.password = TannerConfig.get("POSTGRES", "password")
+        self.db_name = TannerConfig.get("POSTGRES", "db_name")
+        self.poolsize = TannerConfig.get("POSTGRES", "poolsize")
+        self.timeout = TannerConfig.get("POSTGRES", "timeout")
+
+    async def get_pg_client(self):
+        pg_client = None
+        try:
+            dsn = "dbname={} user={} password={} host={} port={}".format(
+                self.db_name, self.user, self.password, self.host, self.port
+            )
+
+            pg_client = await asyncio.wait_for(
+                aiopg.create_pool(dsn, maxsize=self.poolsize), timeout=int(self.timeout)
+            )
+
+            async with pg_client.acquire() as conn:
+                async with conn.cursor() as cur:
+                    await cur.execute(
+                        """
+                    CREATE TABLE IF NOT EXISTS "Paths" (
+                        "sess_uuid" TEXT PRIMARY KEY NOT NULL UNIQUE,
+                        "path" TEXT NOT NULL,
+                        "timestamp" FLOAT NOT NULL,
+                        "response_status" INT NOT NULL,
+                        "attack_type" TEXT NOT NULL
+                    )
+                    """
+                    )
+
+                    await cur.execute(
+                        """
+                    CREATE TABLE IF NOT EXISTS "Cookies" (
```

Comment:
please make the table's name either all with first capital letter or all lowercase :D